contents/careers/senior-product-designer.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Senior Product Designer
+sidebar: Careers
+showTitle: true
+---
+
+<h5 class='centered'>PostHog empowers customers to create successful products.</h5>
+
+We’re looking for a **Senior Product Designer** to join our small (but growing), fully-remote [team](/handbook/company/team/).
+
+Until now, tools for building products have been very fragmented - product analytics, heatmaps, session recording and A/B testing all provide immense value, but existing tools weren’t natively built to work together.
+
+PostHog is different by offering these tools (and more) in a single platform that customers can host themselves.
+
+### What you'll be doing
+We're looking for a seasoned product designer with a background in interaction design, analytics and data visualization to work on the open source PostHog web app.
+
+This is our first dedicated product design hire, joining our [Design Lead](/handbook/company/team#cory-watilo-lead-designer) and [Graphic Designer](/handbook/company/team#lottie-coxon-graphic-designer) whose current efforts are largely focused on our website, docs and contributor experience.
+
+You can expect to do a lot of normal product designer-y things, but with one big twist: *YOU decide what you're going to work on!* We hire high-performing individuals who can identify a problem, propose a solution, and execute on it. We ship every two weeks.
+
+You'll work with our contributors to identify the highest leverage opportunities to improve the UX, grow retention, and assist engineers in designing new features. (Our engineers are incredibly UX-minded, so your suggestions won't fall on deaf ears!)
+This role encompasses everything from building a design roadmap, user research and testing, evolving our library of Figma components, and prototyping interactions.
+
+There's no requirement around how long you've worked in product design, though most designers in this type of role typically have 5-10 years experience.
+
+### What you'll bring
+
+- A design generalist, with a strong opinion about UX best practices
+- A background in analytics and data visualization
+- Strong knowledge of Figma
```

Comment:
Let's change this from ~Strong knowlegde of Figma~ to "Moderate to advanced experience working in Figma"